analysis/ExplorerProperties.py

```diff
@@ -1,19 +1,24 @@
 import math
 
+import PhysicsConstants as PC
+
 def DetectorRadius():
   return 393
 
 def CrystalVolume():
   return 0.276 * 0.276 * 1.81
 
-def CrystalMass():
-  return CrystalVolume() * 7.1
+def CrystalMass( DetectorMaterial ):
+  varName = f'density_{DetectorMaterial}'
+  if varName not in PC.__dict__:
+    raise RuntimeError(f'{varName} does not exist in PhysicsConstants. Wrong material name?')
+  return CrystalVolume() * eval(f'PC.{varName}')
```

Comment:
Surely there is a less cryptic way to write this?

Reply:
I saved densities in PhysicsConstants.py as a dictionary and get the density value from it based on a key which is a string with crystal name. Hopefully this simplifies the code now.